va/va.h

```diff
@@ -1754,9 +1754,13 @@ typedef struct _VAEncPackedHeaderParameterBuffer {
     uint32_t                bit_length;
     /** \brief Flag: buffer contains start code emulation prevention bytes? */
     uint8_t               has_emulation_bytes;
+    /** \brief Minimum offset for emulation prevention bytes insertion,
+     * bytes before the offset will not include emulation prevention bytes.
+     */
+    uint32_t                emulation_bytes_insertion_minimum_offset;
```

Comment:
I'm guessing that for compatibility reasons this value is only used if has_emulation_bytes is zero?  Maybe clarify that.

Reply:
We know that if 0x00 0x00 0x01 occurs inside the slice data, encoder should insert 0x03 in front of 0x01. Application packs SPS, PPS, and slice header, and it sets corresponding VAEncPackedHeaderParameterBuffer. If application doesn't insert 0x03 for NALU start codes emulation prevention, it should set has_emulation_bytes to 0. And then driver will take charge of insertion. There are two cases, one is only one NALU (SPS/PPS/slice header) is included in VAEncPackedHeaderParameterBuffer, then emulation prevention byte (0x03) insertion should be after the first three bytes (0x00 0x00 0x01) at least. The other case is that when two NALUs (SVC prefix NALU + slice header) is in one VAEncPackedHeaderParameterBuffer, then emulation prevention byte should be inserted after at least the whole SVC prefix NALU and the second NALU start codes. The new added emulation_bytes_insertion_minimum_offset is used for both cases, which indicates how many bytes should be skipped to insert 0x03.

Reply:
update the comment for clarifying when emulation_bytes_insertion_minimum_offset is used.